docs/uml/main/DataSourceClassDiagramm_draft.puml

```diff
@@ -0,0 +1,126 @@
+@startuml
+
+note "Assuming all classes to implement \nthe abstract methods of their interfaces\n\n" as generalNotes
+
+interface Publisher
```

Comment:
do we still need this draft?

Reply:
Wasn't sure, because @mia-krause is partly working on it. Therefore, I would like to keep it.